### ISSUE TITLE
Parse errors creating TAP Extended Test Results on Windows

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#3563]
https://ci.eclipse.org/openj9/job/Test-extended.functional-JDK8-win_x86-64_cmprssptrs/64/tapResults/
https://ci.eclipse.org/openj9/job/Test-extended.functional-JDK8-win_x86/62/tapResults/

```
02:40:05         cmdLineTester_jep178_staticLinking_SE80_0_FAILED
02:40:05     duration_ms: 516
02:40:05 ]: while scanning an alias
02:40:05  in 'string', line 14, column 1:
02:40:05     ***
02:40:05     ^
02:40:05 expected alphabetic or numeric character, but found *
02:40:05  in 'string', line 14, column 2:
02:40:05     ***
02:40:05      ^
02:40:05 
02:40:05 	at org.tap4j.parser.Tap13Parser.parseDiagnostics(Tap13Parser.java:435)
02:40:05 	at org.tap4j.parser.Tap13Parser.parseLine(Tap13Parser.java:262)
02:40:05 	at org.tap4j.parser.Tap13Parser.parseTapStream(Tap13Parser.java:224)
02:40:05 	... 15 more
02:40:05 Caused by: while scanning an alias
02:40:05  in 'string', line 14, column 1:
02:40:05     ***
02:40:05     ^
02:40:05 expected alphabetic or numeric character, but found *
02:40:05  in 'string', line 14, column 2:
02:40:05     ***
02:40:05      ^
02:40:05 
02:40:05 	at org.yaml.snakeyaml.scanner.ScannerImpl.scanAnchor(ScannerImpl.java:1439)
02:40:05 	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchAlias(ScannerImpl.java:918)
02:40:05 	at org.yaml.snakeyaml.scanner.ScannerImpl.fetchMoreTokens(ScannerImpl.java:366)
02:40:05 	at org.yaml.snakeyaml.scanner.ScannerImpl.checkToken(ScannerImpl.java:226)
02:40:05 	at org.yaml.snakeyaml.parser.ParserImpl$ParseBlockMappingKey.produce(ParserImpl.java:557)
02:40:05 	at org.yaml.snakeyaml.parser.ParserImpl.peekEvent(ParserImpl.java:157)
02:40:05 	at org.yaml.snakeyaml.parser.ParserImpl.checkEvent(ParserImpl.java:147)
02:40:05 	at org.yaml.snakeyaml.composer.Composer.composeMappingNode(Composer.java:224)
02:40:05 	at org.yaml.snakeyaml.composer.Composer.composeNode(Composer.java:155)
02:40:05 	at org.yaml.snakeyaml.composer.Composer.composeDocument(Composer.java:122)
02:40:05 	at org.yaml.snakeyaml.composer.Composer.getSingleNode(Composer.java:105)
02:40:05 	at org.yaml.snakeyaml.constructor.BaseConstructor.getSingleData(BaseConstructor.java:120)
02:40:05 	at org.yaml.snakeyaml.Yaml.loadFromReader(Yaml.java:450)
02:40:05 	at org.yaml.snakeyaml.Yaml.load(Yaml.java:369)
02:40:05 	at org.tap4j.parser.Tap13Parser.parseDiagnostics(Tap13Parser.java:430)
02:40:05 	... 17 more
02:40:05 TAP parse errors found in the build. Marking build as UNSTABLE
```

**Comment by pshipton** [comment]
@llxia

**Comment by llxia** [comment]
Is possible filter out the special characters in test output? @renfeiw

**Comment by pshipton** [comment]
https://ci.eclipse.org/openj9/job/Test-extended.functional-JDK11-win_x86-64_cmprssptrs/71/tapResults/

**Comment by renfeiw** [comment]
For the first one, we can reproduce the issue and we will find a proper way to fix it. For the second and third ones, we had a hard time to reproduce the failures. @sophia-guo has tried the two tap files, both of them works fine in her build.

**Comment by renfeiw** [comment]
The fix for windows issue is merged: #3563
